pycheribuild/projects/disk_image.py

```diff
@@ -663,6 +676,39 @@ def make_rootfs_image(self, rootfs_img: Path):
                               yes_no_str="")
             raise
 
+    def make_zfs_rootfs_image(self, rootfs_img: Path):
+        try:
+            # Create a single pool with a single FS mounted at /.
+            # 5GiB is currently enough space to install llvm-base, gdb, and
+            # vim on a morello-purecap image with a useful amount of free
+            # space remaining.  XXX: make this tunable
+            self.run_cmd([self.makefs_cmd] + [
+                "-t", "zfs",
+                "-o", "poolname=zroot,rootpath=/,bootfs=zroot",
+                "-s", "5g",
```

Comment:
Aside from comments, these are the only lines that aren't shared with make_ufs_rootfs_image; I think it'd be better to have one combined function like for the mking functions